pkg/controller/login/account.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package login defines the controller for the login page.
+package login
+
+import (
+	"net/http"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/controller"
+)
+
+func (c *Controller) HandleAccountSettings() http.Handler {
```

Comment:
Thinking out loud... We could expand this to allow realm admins to view properties of another user by passing in an ID or qs param.

Reply:
Well we already have the /users/{id} page. This one is intended to show some properties (like password and email status) that are more personal to your auth. I also think its annoying to find myself in the Users index list (which I'm sure will get larger) and it seems like a common task.
There's some overlap though.

One question: do we want "leave realm" and "delete account" options in here

Reply:
I think we want them, but we want extra confirmation on them.